docs/src/drivers/gm.adoc

```diff
@@ -834,35 +815,39 @@ Note that on the Teach Pendant module it cannot be changed, and pre-programmed a
 Upon request it can be delivered with firmware pre-programmed different ID.
 
 .Teach Pendant module pins
-[width="80%",options="header",cols="<3,^2,<6"]
+[width="100%",options="header",cols="<3,^2,<6"]
 |===
-| Pins                | Type and direction | Pin description
-| .adc-<0-5>          | (float, Out)       | Value of ADC input in Volts.
-| .enc-reset          | (bit, In)          | When True, resets counts and position to zero.
-| .enc-counts         | (s32, Out)         | Position in encoder counts.
-| .enc-rawcounts      | (s32, Out)         | The raw count is the counts, but unaffected by reset.
-| .enc-position       | (float, Out)       | Position in scaled units (=.enc-counts/.enc-position-scale).
-| .in-<0-7>           | (bit, Out)         | Input pin
-| .in-not-<0-7>       | (bit, Out)         | Negated input pin
+| Pins             | Type and direction | Pin description
+m| .adc-_<0-5>_    | (float, Out)       | Value of ADC input in Volts.
+m| .enc-reset      | (bit, In)          | When True, resets counts and position to zero.
+m| .enc-counts     | (s32, Out)         | Position in encoder counts.
+m| .enc-rawcounts  | (s32, Out)         | The raw count is the counts, but unaffected by reset.
+m| .enc-position   | (float, Out)       | Position in scaled units (=.enc-counts/.enc-position-scale).
+m| .in-_<0-7>_     | (bit, Out)         | Input pin
+m| .in-not-_<0-7>_ | (bit, Out)         | Negated input pin
 |===
 
 .Teach Pendant module parameters
-[width="80%",options="header",cols="<3,^2,<6"]
+[options="header",cols="<3,^2,<6"]
 |===
 | Parameters            | Type and direction | Parameter description
-| .adc-scale-<0-5>      | (float, R/W)       | The input voltage will be multiplied by scale before being output to .adc- pin.
-| .adc-offset-<0-5>     | (float, R/W)       | Offset is subtracted from the hardware input voltage after the scale multiplier has been applied.
-| .enc-position-scale   | (float, R/W)       | Scale in per length unit.
+m| .adc-scale-_<0-5>_   | (float, R/W)       | The input voltage will be multiplied by scale before being output to .adc- pin.
+m| .adc-offset-_<0-5>_  | (float, R/W)       | Offset is subtracted from the hardware input voltage after the scale multiplier has been applied.
+m| .enc-position-scale  | (float, R/W)       | Scale in per length unit.
 |===
 
 .HAL example
-
-----
-gm.0.rs485.0.adc-0 # First analogue channel of the node.
-# gm.0     - Means the first GM6-PCI motion control card (PCI card address = 0)
-# .rs485.0 - Select node with address 0 on the RS485 bus
-# .adc-0   - Select the first analogue input of the module
-----
+[cols="<3,<6"]
+|===
+| gm.0.rs485.0.adc-0 | First analogue channel of the node.
+|===
+constructed from
+[cols="<3,<6"]
+|===
+m| gm.0     | Means the first GM6-PCI motion control card (PCI card address = 0)
+m| .rs485.0 | Select node with address 0 on the RS485 bus
+m| .adc-0   | Select the first analogue input of the module
+|===
```

Comment:
```suggestion
.First analog channel on RS485 node 0 on card 0
[source,{hal}]
----
gm.0.rs485.0.adc-0 # First analogue channel of the node.
                   # gm.0 - Means the first GM6-PCI motion control card (PCI card address = 0)
                   # .rs485.0 - Select node with address 0 on the RS485 bus
                   # .adc-0 - Select the first analogue input of the module
----

Reply:
Hm. I then prefer the original.

Reply:
What do you think of this: https://github.com/LinuxCNC/linuxcnc/pull/2037#discussion_r980201898?

Reply:
> Hm. I then prefer the original.

Not having it indented makes it unobvious that these lines pertains to the first one because they don't differ much from another commented line.

Reply:
> #2037 (comment)

The indented one is very nice, indeed. A conditione sine qua non for me is that code-blocks can be copied in toto and are executable. So, the non-executable parts need to be commented "away". You seem to have addressed that, so I am fine with it.

Reply:
I think to have addressed this now.